baseplate/crypto.py

```diff
@@ -124,7 +121,7 @@ def make_signature(secret, message, max_age):
     return base64.urlsafe_b64encode(header + digest)
 
 
-def validate_signature(secret, message, signature):
+def validate_signature(secret: "VersionedSecret", message: str, signature: bytes) -> SignatureInfo:
```

Comment:
I don't think this should need the quotes right?

Reply:
gp :hocho: